kit/ChildSession.cpp

```diff
@@ -1639,6 +1644,28 @@ bool ChildSession::mouseEvent(const StringVector& tokens,
     return true;
 }
 
+bool ChildSession::readOnlyClickEvent(const StringVector& tokens)
+{
+    getLOKitDocument()->setView(_viewId);
+    int x = std::atoi(tokens[1].substr(2).data());
+    int y = std::atoi(tokens[2].substr(2).data());
+
+    char *test = getLOKitDocument()->hyperlinkInfoAtPosition(x, y);
```

Comment:
Please run clang-format on new code. vscode can be configured to run automatically on saving on modified parts only (as opposed to the full file).